rust/migrate-wicked/src/tuntap.rs

```diff
@@ -0,0 +1,39 @@
+use agama_server::network::model;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(default)]
+pub struct Tun {
+    pub owner: Option<u32>,
+    pub group: Option<u32>,
```

Comment:
Are these actually optional? When I omit e.g. `TUNNEL_SET_OWNER=` it seems to still populate `owner` to `0`

Reply:
Hm, you are right. On converting ifcfg to wicked always write the owner and group.

But xml could be used by itself with wicked.
```bash
tw-1:/etc/wicked/ifconfig # wicked ifup tun9
tun9            up
tw-1:/etc/wicked/ifconfig # wicked show-config tun9
<interface origin="wicked:xml:/etc/wicked/ifconfig/tun9.xml">
  <name>tun9</name>
  <control>
    <mode>boot</mode>
  </control>
  <firewall/>
  <tun>
    <group>2</group>
  </tun>
  <link/>
</interface>
tw-1:/etc/wicked/ifconfig # cat /etc/wicked/ifconfig/tun9.xml 
<interface >
  <name>tun9</name>
  <control>
    <mode>boot</mode>
  </control>
  <firewall/>
  <tun>
    <group>2</group>
  </tun>
  <link/>
</interface>
````

And such config get's applied to NetworkManager like this:
```
[connection]
id=tun9
uuid=4d7d662c-afa1-4a2f-ad3e-34d92eb6c105
type=tun
interface-name=tun9

[tun]
group=0

[match]

[ipv4]
method=disabled

[ipv6]
addr-gen-mode=default
method=disabled

[proxy]
```

Reply:
Ah, I see